app/src/ereferrals/java/org/eyeseetea/malariacare/domain/usecase/LoginUseCase.java

```diff
@@ -18,7 +20,7 @@ public LoginUseCase(IAuthenticationManager authenticationManager) {
 
     @Override
     public void execute(Credentials credentials, final Callback callback) {
```

Comment:
Login Use Case should has all login steps:
1 - Hardcoded login
2- Get OrgUnit y map to server credentials
3 - Verify user credentials with server credentials